### PR TITLE
New config loader which loads config from specified file path

### PR DESCRIPTION
**Pull request by vkorobkov**
A part of https://github.com/vkorobkov/jfixtures/issues/168

New simplified version of config loader which loads config from a specified file path

### CODE REVIEW

File: src/main/java/com/github/vkorobkov/jfixtures/config/ConfigLoader.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        return new Root(loadNode(path));
+    }
+
+    private Node loadNode(Path path) {
+        if (Files.exists(path)) {
+            return Node.root(YmlUtil.load(path));
+        } else {
+            log.info("File '" + path + "' not found, using default configuration");
+            return Node.emptyRoot();
+        }
```
Comment (rodionovsasha) [inline src/main/java/com/github/vkorobkov/jfixtures/config/ConfigLoader.java]: How about this variant?

```
if (Files.exists(path)) {
      return Node.root(YmlUtil.load(path));
}

log.info("File '" + path + "' not found, using default configuration");
return Node.emptyRoot();
```

File: src/test/resources/config/config_loader_test/default.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+"/default":
+  ".conf.yml":
+    refs:
+      users:
+        avatar_id: images
+      "content.comment":
+        ticket_id: tickets
```
Comment (rodionovsasha) [inline src/test/resources/config/config_loader_test/default.yml]: Add a new line

Comment (vkorobkov) [reply src/test/resources/config/config_loader_test/default.yml]: I added one but github does not display that 😢

Comment (rodionovsasha) [reply src/test/resources/config/config_loader_test/default.yml]: no worries

**Review by rodionovsasha — APPROVED** [review]

**Pull request merged**